modules/mod_translation/lib/css/mod_translation.css

```diff
@@ -0,0 +1,145 @@
+.dropdown-menu.mod_translation_menu-has-icons > li > a {
+  padding-left: 40px;
```

Comment:
That file is generated. Presumably the LESS to CSS script does not use the editorconfig settings.